src/service/invoice-service.ts

```diff
@@ -106,9 +117,24 @@ export default class InvoiceService {
       amount: invoiceEntries.amount,
       priceInclVat: invoiceEntries.priceInclVat.toObject(),
       vatPercentage: invoiceEntries.vatPercentage,
+      custom: true,
```

Comment:
Why is `custom` always true? We no longer support custom invoice entries, right?

Reply:
I will remove them later, as this would blow up the scope of this PR

(requires changing seeder, pdf gen, etc.)